### PR TITLE
base64: add Dutch translation

### PR DESCRIPTION
**Pull request by JenniX3**
- N/A The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.nl/common/base64.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- Decodeer een bestand:
+
+`base64 -d {{bestandsnaam}}`
+
+- Codeer van `stdin`:
+
+`{{eencommando}} | base64`
+
+- Decodeer `stdin`:
```
Comment (navarroaxel) [inline pages.nl/common/base64.md:17]: I don't speak the language, but `van` is not required here?
```suggestion
- Decodeer van `stdin`:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @JenniX3, thanks. Just 1 comment.

File: pages.nl/common/base64.md (hunk 1 above)
Comment (JenniX3) [reply pages.nl/common/base64.md:17]: In fact, you are correct on that one. I will fix this right away.

Comment (navarroaxel) [reply pages.nl/common/base64.md:17]: So `van` is not as translation for `from`?

Comment (JenniX3) [reply pages.nl/common/base64.md:17]: `van` does mean `from`; for the above it is not incorrect, but it is not strictly _required._

**Review by navarroaxel — APPROVED** [review]

**Review by schneiderl — APPROVED** [review]
Don't know the language but looks good.

**Pull request merged**